R/resample.R

```diff
@@ -95,19 +94,48 @@ fit_resamples.formula <- function(formula,
                                   metrics = NULL,
                                   control = control_resamples()) {
 
+  lifecycle::deprecate_soft("0.0.2",
+                            what = "fit_resamples.formula()",
+                            details = deprecate_msg(match.call(), "fit_resamples"))
   empty_ellipses(...)
 
-  if (is_missing(model) || !inherits(model, "model_spec")) {
-    rlang::abort("`model` should be a parsnip model specification object.")
+  fit_resamples(model, preprocessor = formula, resamples = resamples,
+                metrics = metrics, control = control)
+}
+
+#' @export
+#' @rdname fit_resamples
+fit_resamples.model_spec <- function(object,
+                                     preprocessor,
+                                     resamples,
+                                     ...,
+                                     metrics = NULL,
+                                     control = control_resamples()) {
+
+  if (is_missing(preprocessor) || !(inherits(preprocessor, "recipe") || inherits(preprocessor, "formula"))) {
```

Comment:
Try to keep the lines within 80 char. You can use `styler` or cmd-shift-a. I often override those changes (just ask Davis) but keep the lines within the bounds.

Reply:
Addressed in 9ca530512cb153a40454f439161d75f49180e167